tsconfig.json

```diff
@@ -10,7 +11,7 @@
     // output .js.map sourcemap files for consumers
     "sourceMap": true,
     // match output dir to input dir. e.g. dist/index instead of dist/src/index
-    "rootDir": "./typechain",
+    "rootDir": "./",
```

Comment:
Yep it will but can easily make a build-tsconfig.json that inherits the main one and just overwrites the root dir back to type chain and then point the build system to use that

Reply:
That'd be really helpful. The reason I changed `rootDir` to `./` was to silence error messages in `/test` files, which are written in typescript.